cmd/authctl/user/enable.go

```diff
@@ -0,0 +1,34 @@
+package user
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// EnableCmd is a command to enable a user.
+var EnableCmd = &cobra.Command{
```

Comment:
Any need for being exported?

Reply:
I am a golang newbie. I though I would need that to be able to use this command in users.go. I can move this whole subcommand to users.go if we want to avoid exporting this. Or I will be happy to do anything else you would suggest.

Reply:
no need to move it, can just be named with a lower case initial.

Reply:
Done. Thanks.